src/common/KernelBase.cpp

```diff
@@ -631,13 +632,16 @@ void KernelBase::setCaliperMgrVariantTuning(VariantID vid,
   }
   )json";
 
-  if(!ran_spot_config_check && (!addToConfig.empty())) {
+  if(!ran_spot_config_check && ((!addToSpotConfig.empty()) || (!addToCaliConfig.empty()))) {
     cali::ConfigManager cm;
-    std::string check_profile = "spot()," + addToConfig;
+    std::string check_profile = "spot(" + addToSpotConfig + ")";
```

Comment:
Should this check if its empty?

Reply:
It doesn't need to since `spot()` is a valid config and we are always using spot in the config. However I still rewrote this to make it more readable [1190af1](https://github.com/LLNL/RAJAPerf/pull/459/commits/1190af131bfe281aa899f6ae0a156cc8097b6631)